ner_v2/detectors/numeral/number/en/data/numerals_constant.csv

```diff
@@ -33,3 +33,5 @@ number,name_variants,number_value,number_type
 100000,million|mil|m,1000000,scale
 10000000,crore|crores|c|cr,10000000,scale
 100000000,billion|bil|b,1000000000,scale
+11,double,11,scale
```

Comment:
Let me know if I am getting something wrong here, if a user writes double two or double three, it will replace it with two ones and a two here right? Should we handle the double and triple logic differently by incrementing the count of the number appearing after it?

Reply:
Double one should give 11, this is handled in Number Detector logic. Will be adding comments for better understanding there